### PR TITLE
Add base64 converter for chapter name in mediainfo

### PR DESCRIPTION
**Pull request by harshithmohan**
Not sure I wrote it properly.. but it works<sup>TM</sup>

### CODE REVIEW

**Comment by harshithmohan** [comment]
I was not able to use the specific type to just target the object with @dt and #value

File: Shoko.Server/Utilities/MediaInfoLib/MenuBase64Converter.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        sanitizedDictionary[key] = value?.ToString();
+                        break;
+                    case JTokenType.Object:
+                    {
+                        var base64EncodedBytes = Convert.FromBase64String(value?["#value"]?.ToString() ?? "");
+                        sanitizedDictionary[key] = Encoding.UTF8.GetString(base64EncodedBytes).Trim();
+                        break;
+                    }
+                    default:
+                        throw new NotImplementedException();
```
Comment (harshithmohan) [inline Shoko.Server/Utilities/MediaInfoLib/MenuBase64Converter.cs:41]: Was not sure what to put for default case.

Comment (Cazzar) [reply Shoko.Server/Utilities/MediaInfoLib/MenuBase64Converter.cs:41]: Throw something more specific then a not implemented description, or at least provide a message

Comment (da3dsoul) [reply Shoko.Server/Utilities/MediaInfoLib/MenuBase64Converter.cs:41]: If it's intentionally not implemented, I tend to go for a NotSupportedException

Comment (harshithmohan) [reply Shoko.Server/Utilities/MediaInfoLib/MenuBase64Converter.cs:41]: I changed it to
`throw new NotSupportedException("Unknown type encountered in MenuStream.extra");`

**Pull request merged**